scripts/st2bootstrap-el7.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+
+set -eux
```

Comment:
Please don't set -x. In fact don't set anything unless needed. If installation fails at a step and I re-run the script, I get 

```
[vagrant@st2test ~]$ ./st2bootstrap-el7.sh
+ adjust_selinux_policies
+ grep -q Enforcing
+ getenforce
+ install_st2_dependencies
+ sudo yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
Loaded plugins: fastestmirror
epel-release-latest-7.noarch.rpm                                                                                                                                                                                                      |  14 kB  00:00:00
Examining /var/tmp/yum-root-NM4sll/epel-release-latest-7.noarch.rpm: epel-release-7-5.noarch
/var/tmp/yum-root-NM4sll/epel-release-latest-7.noarch.rpm: does not update installed package.
Error: Nothing to do
[vagrant@st2test ~]
```